doc/rbd/rados-rbd-cmds.rst

```diff
@@ -28,26 +28,29 @@ Create a Block Device Pool
 Create a Block Device User
 ==========================
 
-Unless specified, the ``rbd`` command will access the Ceph cluster using the ID
-``admin``. This ID allows full administrative access to the cluster. It is
-recommended that you utilize a more restricted user wherever possible.
+Unless otherwise specified, the ``rbd`` command uses the ID ``admin`` to access the Ceph cluster. The ``admin`` ID allows full administrative access to the cluster. We recommend that you acess the Ceph cluster with an ID that has fewer permissions than the ``admin`` ID does. We call this non-``admin`` ID a "block device user" or "Ceph user".
```

Comment:
Instead of bare ID, I'd use "user ID" everywhere.

Reply:
I might suggest "Ceph user" instead, I think "user ID" is even more likely to be confused with Linux /etc/passwd-type users.

Reply:
I'm going to go with "Ceph user ID" for now, for minimum ambiguity.